addons/sitting/functions/fnc_sit.sqf

```diff
@@ -0,0 +1,40 @@
+/*
+ * Author: Jonpas
+ * Sits down the player.
+ *
+ * Arguments:
+ * 0: Seat <OBJECT>
+ * 1: Player <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [seat, player] call ace_sitting_fnc_sit;
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+PARAMS_2(_seat,_player);
+
+// Set global variable for standing up
+GVAR(seat) = _seat;
+
+// Overwrite weird position, because Arma decides to set it differently based on current animation/stance...
+_player switchMove "amovpknlmstpsraswrfldnon";
+
+// Read config
+_sitDirection = getNumber (configFile >> "CfgVehicles" >> typeOf _seat >> QGVAR(sitDirection));
+_sitPosition = getArray (configFile >> "CfgVehicles" >> typeOf _seat >> QGVAR(sitPosition));
+
+// Set direction and position
+_player setDir ((getDir _seat) + _sitDirection);
+_player setPos (_seat modelToWorld _sitPosition);
```

Comment:
Just tried, `setPos` works fine above terrain level as well since it uses modelToWorld. Is it worth calling `positionToASL` with `setPosASL` if it works?

I did notice it doesn't apply bitch and bank though, will do that.